Sources/SwiftyRequest/RestRequest.swift

```diff
@@ -740,3 +757,31 @@ public enum RestError: Error, CustomStringConvertible {
         }
     }
 }
+
+// URL Session extension
+extension RestRequest: URLSessionDelegate {
+    
+    /// URL session function to allow trusting certain URLs
+    public func urlSession(_ session: URLSession, didReceive challenge: URLAuthenticationChallenge, completionHandler: @escaping (URLSession.AuthChallengeDisposition, URLCredential?) -> Void) {
+        let method = challenge.protectionSpace.authenticationMethod
+        let host = challenge.protectionSpace.host
+        switch (method, host) {
+        case (NSURLAuthenticationMethodServerTrust, self.url):
+            #if MAC_OS_X_VERSION_10_6
+                if let trust = challenge.protectionSpace.serverTrust {
+                    let credential = URLCredential(trust: trust)
+                    completionHandler(.useCredential, credential)
+                } else {
+                    Log.debug("Attempting to establish a secure connection; no server trust established. Resorting to default handling.")
+                    completionHandler(.performDefaultHandling, nil)
+                }
+            #else
+                Log.debug("Attempting to establish a secure connection; macOS 10.6 or higher must be used to achieve this. Resorting to default handling.")
```

Comment:
@ShihabMehboob Can we change the log level from debug to warning in these lines (775, 779)?

Reply:
bump